webapp/client/src/components/FormHeader.js

```diff
@@ -1,15 +1,12 @@
 import PropTypes from "prop-types";
-import styled from "styled-components";
-
-const Intro = styled.p`
-  font-size: var(--text-medium);
-`;
+import FormHeaderIntro from "./FormHeaderIntro";
 
 export default function FormHeader({ title, intro, hideIntro }) {
+  const paragraphs = [].concat(intro);
   return (
     <div>
       <h1 className="my-4">{title}</h1>
-      {intro && !hideIntro && <Intro>{intro}</Intro>}
+      {intro && !hideIntro && <FormHeaderIntro paragraphs={paragraphs} />}
```

Comment:
Could you add a test for hideIntro?

Reply:
We had a test for hideIntro=True. I have added one for hideIntro=False.